patternfly/patternfly-elements
# fix: Add support for active vs. inactive background colors on wind variant of tabs

## pfe-tabs

- Add CSS variable support for  active vs inactive background colors on wind variant
- Add CSS variable support for typography (and fix so vars are empty by default)
- Update documentation with correct variables for this component (only)

### Related issue

https://github.com/patternfly/patternfly-elements/issues/1123


### Preview

- [demo page where this element can be viewed](https://deploy-preview-1124--happy-galileo-ea79c4.netlify.app/elements/pfe-tabs/demo/) 
- [alt demo page](https://deploy-preview-1124--happy-galileo-ea79c4.netlify.app/elements/pfe-tabs/demo/all-combos.html?my-tabs=tab2)


### Testing instructions

1. Go to the pfe-tabs demo page 
2. add these style overrides
```
:root {
    --pfe-tabs--BackgroundColor: pink !important;
    --pfe-tabs--BackgroundColor--inactive: beige !important;
    --pfe-tabs__tab--FontSize: 40px;
    --pfe-tabs__tab--FontWeight: 900;
    --pfe-tabs__tab--LineHeight: 2.5;
}
```
3.  Only the active tabs (earth & wind) should inherit the (active) background color (pink), the other tabs should inherit the inactive background color (beige). Font size of the tabs should become 40px, with a 2.5 line height and a weight of 900.

#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
